### DISCUSSION TITLE
TSForecastingExperiment Input

### DISCUSSION THREAD

**Discussion by rreube3** [pycaret/pycaret#2682] [Q&A]
I wanted to know if I should be passing non-stationary time series data into the TSForecastingExperiment function or using differencing to convert to stationary data and pass that.  From what I can tell using both of these methods yields different results in terms of recommended optimal models (compare_models function) so any help would be very appreciated regarding which dataset to pass.

**Comment by ngupta23** [comment]
All models in PyCaret can handle non stationarity. You can pass the no stationary data directly.
